lib/view/acknowledgements_view.dart

```diff
@@ -67,6 +67,33 @@ Several companies who have provided support and employee time, including <a href
 <br/>
 <br/>
 Those who have given helpful and timely advice, including Byron Teu.<br/>
+  ''';
+  static const _contributorsHtml = '''
+<b>Gregg Miller</b> - Initial Development &amp; Ideas<br/>
+<b>Matt Kelsh</b> - Design<br/>
+<b>Marc Edwards</b> - Design<br/>
+<b>Luke Sleeman</b> - Project Organiser<br/>
+<b>Alessandro Favero</b> - Flutter Dev<br/>
+<b>Maksim Lin</b> - Senior Flutter Dev, CI<br/>
+<b>Mike Hughes</b> - Lead Flutter Dev, CI &amp; CD<br/>
+<b>Adam Koch</b> - Flutter Dev<br/>
+<b>Brett Morgan</b> - Flutter Dev<br/>
+<b>Eric Jiang</b> - Flutter Dev<br/>
+<b>Dylan Lange</b> - Flutter Dev<br/>
+<b>Jonathan Galtieri</b> - Anaesthetist<br/>
+<b>Ben Blackman</b> - Anaesthetic Registrar<br/>
+<b>Fabien Dade</b> - ICU Registrar<br/>
+<b>Alice Gynther</b> - Anaesthetist<br/>
+<b>Kim Nguyen</b> - Flutter Dev<br/>
+<b>Bramley</b> - Flutter Dev<br/>
```

Comment:
@rockgecko-development  - do you want us to add your second name?  What is it?!

Reply:
Yeah, I just copy and paste from Zeplin. Would be good to let everyone check their names.